plugins/synaptics-mst/fu-synaptics-mst-device.c

```diff
@@ -1756,28 +1777,6 @@ fu_synaptics_mst_device_setup(FuDevice *device, GError **error)
 		return FALSE;
 	}
 
-	/* detect chip family */
-	switch (self->family) {
-	case FU_SYNAPTICS_MST_FAMILY_TESLA:
-		fu_device_set_firmware_size_max(device, 0x10000);
-		fu_device_set_firmware_size_max(device, 0x10000);
-		break;
-	case FU_SYNAPTICS_MST_FAMILY_PANAMERA:
-		fu_device_set_firmware_size_max(device, 0x80000);
-		fu_device_add_flag(device, FWUPD_DEVICE_FLAG_DUAL_IMAGE);
```

Comment:
I still think you need the DUAL_BANK info otherwise it's not going to work on ChromeOS.

Reply:
Just add 
`fu_device_add_flag(device, FWUPD_DEVICE_FLAG_DUAL_IMAGE);`
is okay?

Reply:
Yup.